test/e2e/common/lifecycle_hook.go

```diff
@@ -130,7 +173,26 @@ var _ = framework.KubeDescribe("Container Lifecycle Hook", func() {
 			testPodWithHook(podWithHook)
 		})
 		/*
-			Release: v1.9
+			Release : v1.19
+			Testname: Pod Lifecycle, poststart https hook
+			Description: When a post-start handler is specified in the container lifecycle using a 'HttpGet' action, then the handler MUST be invoked before the container is terminated. A server pod is created that will serve https requests, create a second pod with a container lifecycle specifying a post-start that invokes the server pod to validate that the post-start is executed.
+		*/
+		framework.ConformanceIt("should execute poststart https hook properly [NodeConformance]", func() {
```

Comment:
new tests need to start as regular e2e tests and demonstrate non-flakiness before being promoted to conformance tests. cc @kubernetes/cncf-conformance-wg